recipes/ceres-solver/all/conanfile.py

```diff
@@ -47,6 +66,20 @@ def config_options(self):
             del self.options.use_CXX11_threads
             del self.options.use_CXX11
 
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("eigen/3.4.0")
+        if self.options.use_glog:
+            self.requires("glog/0.5.0")
+        if self.options.use_gflags:
+            self.requires("gflags/2.2.2")
+            self.options["gflags"].nothreads = False
```

Comment:
For my learning, is it safe to override default options of dependencies in requirements?

I was under the impression it was best to set that in `config_options`, is that a rule of thumb?
I also don't see a check in validate in the git diff 🤔 is that because of the location?

Reply:
No if you really have to, it's in `configure()`

Reply:
should this be moved then, it looks to be in the requirements right now 🤔

Reply:
yes, I didn't change this part.